daemon.py

```diff
@@ -12,11 +12,7 @@
 def main():
     config = get_config(os.environ)
     session = requests.Session()
-    session.headers = {
-        'Content-Type': 'application/imap-to-webhook-v1+json'
```

Comment:
Wiem, że to było usunięte na `v2`, ale czy możemy to zachować?

Reply:
Nie widzę przeszkód. Już poprawiłem.

Reply:
Zauważam, że `Content-Type` sprawdzamy po stronie Fedrowania. Gwarantuje to, że wiadomość zostanie zrozumiała przez Fedrowanie / Poradnię, albo wiadomość pozostanie na serwerze (w osobnym katalogu), gdzie ją łatwo odtworzyć (przenieść ponownie do `INBOX`).

Czy potrzebujemy podnieść wersje?

Reply:
Jeżeli podnosimy wersje to oznacza, że nie musimy przejmować się kompatybilnością, a więc proponuje zamienić `content` na `plain_content`, tak jak chciałeś.

Reply:
Nie za bardzo rozumiem pytanie. Chodzi o wersję aplikacji? Możemy albo zostać przy `v2` albo stworzyć `v3`. W każdym razie warto zaktualizować `master` do najnowszej wersji.

Reply:
Powinniśmy podnieść wersje formatu danych do `v2`, która jest zawarta w `application/imap-to-webhook-v1+json`, bo skoro zrobiłem osobny branch to zapewne nie jest ona wstecznie kompatybilna.

Reply:
Rozumiem, w takim razie ustawię nagłówek na `application/imap-to-webhook-v2+json` a w Fedrowaniu mogę obsłużyć obie wersje.

Reply:
Myślę, że nie warto tracić czasu na obsługę obu wersji. Możemy zsynchronizować aktualizacje, a brak pobierania wiadomości przez godzinę w weekend to nic.